project/redis-master.sls

```diff
@@ -0,0 +1,17 @@
+# Install Redis, listening on all interfaces.
+# This does not open the firewall for Redis, so do that elsewhere
```

Comment:
All the ones I could think of eventually moved to ElasticCache. Simple sounds good to me.